### ISSUE TITLE
sometimes, skript does not allow you to shut down the server

### ISSUE THREAD

**Issue by danielzinh** [SkriptLang/Skript#3896]
### Description
Sometimes, when i close the server, show:

[Skript] Disabling Skript v2.5.2 and doesn't close
![image](https://user-images.githubusercontent.com/50877639/114318895-68b7db00-9ae5-11eb-871b-ddcf4ab65b10.png)

### Steps to Reproduce
i dont know

### Expected Behavior
the server closes, but he stay permanently in "Disabling Skript"

### Errors / Screenshots

nothing

![image](https://user-images.githubusercontent.com/50877639/114318945-9735b600-9ae5-11eb-815c-2d61f29cadfd.png)

### Server Information
* **Server version/platform:** Paperspigot 1.16.5
* **Skript version:** 2.5.2

### Additional Context
this happens ONLY SOME TIMES. and I need to kill the server.

**Comment by TheDGOfficial** [comment]
Any reason you're not using 2.5.3? Do you have a lot of variables? Which Java version you're using? My guess is Skript is either saving variables or cleaning up fields, but you have to enable debug mode to be sure.

Might be a slow disk or Windows issue too, I can recommend using Linux instead of Windows/Windows Server, not only open source/free/bloatware free, it also has netty native transport and such. Even if you want to use Windows, you should not use Server 2012 or 2008 (which I can guess from screenshot), you probably want to use 2016 or 2019, 2008 and 2012 are EOL.

In short, try with 2.5.3 & debug mode and post the results then we can diagnose it further. Confirming it on a different OS will help to ensure this is actually a Skript issue, too.

**Comment by danielzinh** [comment]
> Alguma razão pela qual você não está usando o 2.5.3? Você tem muitas variáveis? Qual versão do Java você está usando? Meu palpite é que o Skript está salvando variáveis ​​ou limpando campos, mas você precisa habilitar o modo de depuração para ter certeza.
> 
> Pode ser um disco lento ou problema do Windows também, posso recomendar o uso do Linux em vez do Windows / Windows Server, não apenas open source / free / bloatware free, mas também tem transporte nativo netty e tal. Mesmo se você quiser usar o Windows, não deve usar o Server 2012 ou 2008 (o que posso adivinhar pela captura de tela), você provavelmente deseja usar 2016 ou 2019, 2008 e 2012 são EOL.
> 
> Resumindo, experimente o 2.5.3 & modo de depuração e publique os resultados para que possamos diagnosticar melhor. A confirmação em um sistema operacional diferente ajudará a garantir que esse seja realmente um problema do Skript também.

i use java 8, and i have +- only 20 variables in my server

**Comment by TheDGOfficial** [comment]
That maybe something else then, did you try to reproduce this with debug mode enabled?

**Comment by iamthewallrus** [comment]
> Any reason you're not using 2.5.3? Do you have a lot of variables? Which Java version you're using? My guess is Skript is either saving variables or cleaning up fields, but you have to enable debug mode to be sure.
> 
> Might be a slow disk or Windows issue too, I can recommend using Linux instead of Windows/Windows Server, not only open source/free/bloatware free, it also has netty native transport and such. Even if you want to use Windows, you should not use Server 2012 or 2008 (which I can guess from screenshot), you probably want to use 2016 or 2019, 2008 and 2012 are EOL.
> 
> In short, try with 2.5.3 & debug mode and post the results then we can diagnose it further. Confirming it on a different OS will help to ensure this is actually a Skript issue, too.


Je n'utilise pas Skript 2,5,3

**Comment by TPGamesNL** [comment]
Closing due to lack of response
